src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -366,14 +412,47 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                     scene.pointerY = this._screenCoordinatesRef.y;
                 }
             }
-            controllerData.pick = this._scene.pickWithRay(controllerData.tmpRay, this._scene.pointerMovePredicate || this.raySelectionPredicate);
 
-            const pick = controllerData.pick;
+            let pick = null;
+            let hoverAtOrigin = false;
+            if(controllerData.hoverIndexMesh) {
+                let hoverInfo = this._pickWithMesh(controllerData.hoverIndexMesh, false, this.nearPickPredicate || this.nearGrabPredicate);
+                let nearHover = hoverInfo && hoverInfo.pickedPoint && hoverInfo.hit;
+                if(hoverInfo?.pickedPoint)
+                {
+                    hoverAtOrigin = hoverInfo.pickedPoint.x == 0 && hoverInfo.pickedPoint.y == 0 && hoverInfo.pickedPoint.z == 0;

```

Comment:
What situation causes this to be true?

Reply:
this happens right at the moment the hand tip spheres are created( they are created at origin and then moved to the position of index tip on OnXRFrame above) and if there is a  mesh that happens to have nearinteraction turned on at origin, this is true automatically even if there was no hover or pick. As we know the hand tip spheres are created every time the hand goes out of the scene and comes back in, this does happen quite a few times